lab-jinho/lib/file-reader.js

```diff
@@ -0,0 +1,21 @@
+'use strict';
+
+const fs = require('fs');
+
+const fileReader = module.exports = function(pathArray, callback){
+  var result = [];
+  fs.readFile(pathArray[0], function(err, data){
+    if (err) return callback(err);
+    result.push(data.toString('hex', 0, 8));
+    fs.readFile(pathArray[1], function(err, data) {
+      if (err) return callback(err);
+      result.push(data.toString('hex', 0, 8));
+      fs.readFile(pathArray[2], function(err, data) {
+        if (err) return callback(err);
+        result.push(data.toString('hex', 0, 8));
+        console.log('result:', result);
+        callback(null, result);
+      });
+    });
+  });
+};
```

Comment:
Good job making use of nested callbacks to handle async- fortunately we won't be dealing with nested callback mayhem anymore. Also, pathArray is just a parameter so your computer will not recognize it as an array. 

You could omit pathArray all together and still get the same result. By nature of nesting the callbacks as you have done here, the files will still read asynchronously :)